src/Trakx.Kaiko.ApiClient.Tests/Integration/ReferenceDataTests.cs

```diff
@@ -17,10 +19,15 @@ public async Task ExchangesClient_should_return_list_of_exchanges()
         response.Content.Data.Should().NotBeNull();
         response.Content.Data.Should().HaveCountGreaterThan(0);
 
-        foreach (var item in response.Content.Data.OrderBy(p => p.Name))
+        var data = response.Content.Data.Select(p => new
         {
-            Output.WriteLine($"[{item.Code}] {item.Name}");
-        }
+            code = p.Code,
+            name = p.Name,
+            kaiko_legacy_slug = p.Kaiko_legacy_slug,
```

Comment:
This "test" and JSON format is useful to allow to copy/paste exchanges to configuration in other projects that use this client.